namespaces/live-1.cloud-platform.service.justice.gov.uk/calculate-release-dates-api-dev/resources/variables.tf

```diff
@@ -0,0 +1,42 @@
+
+variable "cluster_name" {
+}
+
+variable "application" {
+  description = "Name of Application you are deploying"
+  default     = "Calculate release dates API"
+}
+
+variable "namespace" {
+  default = "calculate-release-dates-api-dev"
+}
+
+variable "business_unit" {
+  description = "Area of the MOJ responsible for the service."
+  default     = "HMPPS"
+}
+
+variable "team_name" {
+  description = "The name of your development team"
+  default     = "farsight-devs"
+}
+
+variable "environment" {
+  description = "The type of environment you're deploying to."
+  default     = "dev"
+}
```

Comment:
This is what I meant above about consistent env names - I would use `dev` everywhere to make it simple.

Reply:
That has been addressed